### ISSUE TITLE
Implement new clip download routine

### ISSUE THREAD

**Issue by SReject** [crowbartools/Firebot#635]
The current clip downloading routine for both v4 and v5 is incorrect. To fix, the following new routine needs to be implemented:

```js
const exec = require('child_process').exec;
const ffmpeg = require('ffmpeg-static').path;

// returns a promise: resolves with output file path or rejects with conversion error
module.exports = function (m3u8, output) {
    return new Promise((resolve, reject) => {
        exec(
            `${ffmpeg} -i ${m3u8} -c copy -bsf:a aac_adtstoasc "${output}"`,
            {windowHide: true},
            function (err) {
                if (err) {
                    return reject(err);
                }
                resolve(output);
            }
        );
    });
};
```

### Notes  
m3u8stream will no longer be needed and thus should be removed from dependency  

ffmpeg-static will either need to be installed as a dependency or, depending on how electron's packaging works, its module folder will need to be exported to /resources/ upon compilation.

**Comment by SReject** [comment]
Implemented for v4 as of commit a11330a322aabf76fab211c750fa989ff3656656

**Comment by SReject** [comment]
Fixed as of [https://github.com/crowbartools/Firebot/releases/tag/v4.13.4](v4.13.4)
